AMW_angular/io/src/app/settings/deployment-parameter/deployment-parameter.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="container">
+  <h1>Deployment Parameter Keys</h1>
+  <div class="row mt-2 py-4">
+    <div class="col-6 ms-auto">
+      <div class="input-group">
+        <input
+          id="keyName"
+          type="text"
+          class="form-control"
+          [(ngModel)]="keyName"
+          placeholder="Create new key"
+          (keyup.enter)="addKey()"
+        />
+        <button class="btn btn-primary" (click)="addKey()" [attr.data-cy]="'button-add'">Add</button>
+      </div>
+    </div>
+  </div>
+
+  <table class="table table-bordered">
+    <thead>
+    <tr>
+      <th>Name</th>
+      <th></th>
+    </tr>
+    </thead>
+    <tbody>
+    @for (paramKey of paramKeys; track paramKey) {
+    <tr>
+      <td>{{ paramKey.key }}</td>
```

Comment:
Name and icon should be on the same line.
![2024-02-06_15-38](https://github.com/liimaorg/liima/assets/24672636/99417836-b8de-4b19-83df-8cc7b6241fbe)